packages/notifications-system/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "notifications-system",
```

Comment:
Package names have to match the pattern `@booking-services/your_package_name`, so your package name should be @booking-`services/notifications-system`